### PR TITLE
Fixed swipe to refresh in Contributors page

### LINKED ISSUE #221: Add swipe to refresh in Contributors page.
## Overview (Required)
Title says everything 😉

### PR DESCRIPTION
**Pull request by henteko**
## Issue
- close #221 

## Overview (Required)
- Supported swipe to refresh in contributors page

## Links
-

## Screenshot

[https://gyazo.com/07d0cb8816373456ab128fdb82962c3f](https://gyazo.com/07d0cb8816373456ab128fdb82962c3f)

### CODE REVIEW

**Comment by mhidaka** [comment]
:eyes:

**Comment by mhidaka** [comment]
:+1: LGTM 
Thankyou for your works. great job!
I'll merge after ci build.

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsViewModel.kt [hunk 1]
```diff
@@ -32,6 +32,14 @@ class ContributorsViewModel @Inject constructor(
 
     @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
     fun onCreate() {
+        refreshContributors()
+    }
+
+    fun onRefreshContributors() {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsViewModel.kt]: 👍

**Comment by henteko** [comment]
thankyou!

**Pull request merged**